docs/cloud-provider-config.md

```diff
@@ -287,7 +285,6 @@ For authentication methods, only Service Principal supports this feature, and `a
 The following are the default rate limiting values configured in [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) and [AKS-Engine](https://github.com/Azure/aks-engine) clusters prior to Kubernetes version v1.18.0.
 
 ```json
-    "cloudProviderBackoffMode": "v2",
```

Comment:
Can you add more details in the table to when `v2` should be specified?

Reply:
"v2" is not supported anymore since v1.18 as cloud-provider's own backoff retries have been added.

Reply:
Also, AKS doesn't set this option

Reply:
Can you add this info to the table as such?

Reply:
Good catch, added now.